recipes-core/base-files/base-files_%.bbappend

```diff
@@ -1 +1,14 @@
-FILESEXTRAPATHS_append := "${THISDIR}/base-files:"
\ No newline at end of file
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_append = " \
+file://fstab.append \
+file://fstab_selinux.append \
+"
+do_install_append() {
```

Comment:
Using two files approach is not so convenient. The further modification should be done in both files. Consider to add selinux context with sed or other utility. Or you can have all entries in the file and add aos entries using cat in the recipe. In this case, you can add context using var.

Reply:
Done